auto_gptq/nn_modules/qlinear/qlinear_cuda.py

```diff
@@ -44,6 +45,7 @@ def __init__(
         self.bits = bits
         self.group_size = group_size if group_size != -1 else infeatures
         self.maxq = 2 ** self.bits - 1
+        self.weight_dtype = weight_dtype
```

Comment:
This attribute is probably not needed in that a call to `to()` will not change its value.